runtime/interpreter/value.go

```diff
@@ -6006,6 +6006,90 @@ func (AddressValue) SetMember(_ *Interpreter, _ LocationRange, _ string, _ Value
 	panic(errors.NewUnreachableError())
 }
 
+// AccountKeyValue
+
+type AccountKeyValue struct {
+	publicKey []byte
+}
+
+func NewAccountKeyValue(publicKey []byte) AccountKeyValue {
+	return AccountKeyValue{publicKey: publicKey}
+}
+
+func (AccountKeyValue) IsValue() {}
+
+func (AccountKeyValue) DynamicType(_ *Interpreter) DynamicType {
+	return AccountKeyDynamicType{}
+}
+
+func (v AccountKeyValue) Copy() Value {
+	return v
+}
+
+func (v AccountKeyValue) KeyString() string {
+	return v.String()
+}
```

Comment:
`KeyString` is for `HasKeyString`, which should only be implemented if the value can be used as a key in a dictionary, which isn't the case for an account key.
```suggestion
```